src/components/approval-type/ApprovalType.vue

```diff
@@ -108,6 +113,9 @@ export default class ApprovalType extends Vue {
   /** Whether this section is invalid. */
   @Prop({ default: false }) readonly invalidSection!: boolean
 
+  /** Whether the filing is limited restoration extension. */
+  @Prop({ default: false }) readonly isExtension!: boolean
```

Comment:
This component should not know whether this is an extension -- just rename this prop to something that component can know, like whether to show Court Order section (or whatever).

This is separation of concerns: this component shouldn't know what's going on "above" it. Likewise, the parent doesn't need to know the details of what's going on "below" it.